static/js/util.js

```diff
@@ -284,44 +293,69 @@ export function clean_user_content_links(html) {
         } else {
             elt.removeAttribute("target");
         }
+    }
+    return content.innerHTML;
+}
 
-        const is_inline_image =
-            elt.parentElement && elt.parentElement.classList.contains("message_inline_image");
-        if (is_inline_image) {
-            // For inline images we want to handle the tooltips explicitly, and disable
-            // the browser's built in handling of the title attribute.
-            if (elt.getAttribute("title")) {
-                elt.setAttribute("aria-label", elt.getAttribute("title"));
-                elt.removeAttribute("title");
-            }
+export function initialize() {
+    $("#main_div").on("mouseenter", "a", (e) => {
```

Comment:
I think the right selector should be something like `$("body").on("mouseenter", ".rendered_markdown a",` -- this will make this work naturally in stream descriptions, for example. 

(Actually probably we need a tighter class).